shims/common/src/main/scala/org/apache/gluten/GlutenConfig.scala

```diff
@@ -1249,6 +1251,13 @@ object GlutenConfig {
       .checkValues(Set("none", "auto"))
       .createWithDefault("auto")
 
+  val COLUMNAR_VELOX_MAX_SPILL_LEVEL =
+    buildConf("spark.gluten.sql.columnar.backend.velox.maxSpillLevel")
+      .internal()
+      .doc("The max allowed spilling level with zero being the initial spilling level")
+      .intConf
+      .createWithDefault(6)
```

Comment:
do we really allow this to be set to 0?

Reply:
It seems that a 0 indicates that spill is disabled. It's OK to allow the setting. (0 is the initial spilling level)